cekit/descriptor/execute.py

```diff
@@ -23,13 +23,14 @@ def __init__(self, descriptor, module_name):
         super(Execute, self).__init__(descriptor)
 
         descriptor['directory'] = module_name
-
         descriptor['module_name'] = module_name
 
         if 'name' not in descriptor:
-            logger.warning("No value found for 'name'; using auto-generated value of {}/{}".
-                           format(module_name, descriptor['script']))
-            descriptor['name'] = "%s/%s" % (module_name, descriptor['script'])
+            # Generated name
+            descriptor['name'] = "{}/{}".format(module_name, descriptor['script'])
+
+            logger.debug("No value found for 'name' key in the execute section of the '{}' module; using auto-generated value: '{}'".format(
```

Comment:
Why debug? I thought we were trying to warn people?

Reply:
Yes, but not necessarily for names in the scripts, which is **very** unlikely to be overriden. Nobody is defining a name for it, I would call it internal.

Reply:
In other words - printing a warning for the missing `name` key in the execute section of the module doesn't make sense. We can print it, but as a debug. Main use case is that modules are overridden as a whole and it does not make much sense to override particular execute scripts.

All modules define install scripts without defining the name, depending on the image complexity we would end up with many warnings, where fixing it would not give us anything because of the reason above.